docs/data/quality-metrics.md

```diff
@@ -0,0 +1,80 @@
+---
+id: qc-metrics
+title: Quality Control Metrics
+sidebar_label: QC Metrics
+---
+
+ICGC ARGO provides access to data from ARGO member programs generated through the standardized [ARGO Analysis pipelines](/docs/analysis_workflows/data-types). The following page lists quality control metric analyses and data files generated by the ARGO Analysis pipelines.
+
+Quality control metrics are collected and recorded at several checkpoints in the ARGO Analysis pipeline to ensure that only high-quality data is released. All QC Metrics are associated to a corresponding analysis/file set that they annotate.
+
+## Read Group QC
+
+File containing read group (lane) level QC metrics.
+
+### File Types
+
+| Filename Pattern       | Description                                    | Analysis Type | Data Category           | Generating Workflow(s) |
+| ---------------------- | ---------------------------------------------- | ------------- | ----------------------- | ---------------------- |
+| \*.ubam_qc_metrics.tgz | Generated by Picard CollectQualityYieldMetrics | qc_metrics    | Quality Control Metrics | DNA Seq Alignment      |
+
+## Alignment QC
+
+Files containing quality control metrics for aligned CRAM files.
+
+### File Types
+
+| Filename Pattern   | Description                             | Analysis Type | Data Category           | Generating Workflow(s)                                 |
+| ------------------ | --------------------------------------- | ------------- | ----------------------- | ------------------------------------------------------ |
+| \*.qc_metrics.tgz  | Generated by Samtools stats             | qc_metrics    | Quality Control Metrics | DNA Seq Alignment                                      |
+| \*.bas_metrics.tgz | Generated by Sanger bam_stats.pl script | qc_metrics    | Quality Control Metrics | Sanger WGS Variant Calling, Sanger WXS Variant Calling |
```

Comment:
@rosibaj , a minor comment. do you think it is easy to read by separating different workflows with `enter`?

Reply:
@lindaxiang hmm yess i will try and see if ther eis an accepted markdown configuration docusaurus to do this

Reply:
i tried a few things, but cant get this right now. ill continue to investigate for future.

Reply:
No worries Rosi. This is not critical at all. We can get back to it anytime later if needed.